api/api/views/sandbox_survey_view.py

```diff
@@ -1,12 +1,51 @@
-
+from django.utils import timezone
 from rest_framework import permissions, generics
+from rest_framework.response import Response
+from api.models.sandbox_survey import SandboxSurvey
+from django.http import HttpResponseBadRequest
+import json
+
 
 class SandboxSurveyView(generics.GenericAPIView):
-    permission_classes = (permissions.IsAuthenticated,)
-
-    def get(self, request, name=None):
-        "hello"
-        
-    def put(self, request, pk, name=None):
-        "hello"
-    
+    # permission_classes = (permissions.IsAuthenticated,)
+
+    def get(self, request):
+        name = request.GET["sandbox_name"]
+        sandbox = SandboxSurvey.objects.get(sandbox_name=name)
+        data = {
```

Comment:
Try using a serializer for this: 
https://www.django-rest-framework.org/api-guide/serializers/

Reply:
https://github.com/bcgov/representation-grant-app/blob/main/api/api/serializers.py